pkg/rpcclient/invoker/invoker.go

```diff
@@ -137,6 +137,18 @@ func (h *historicConverter) TraverseIterator(sessionID, iteratorID uuid.UUID, ma
 	return h.client.TraverseIterator(sessionID, iteratorID, maxItemsCount)
 }
 
+// Signers returns the set of current invoker signers which is mostly useful
+// when working with upper-layer actors. Returned slice is a newly allocated
+// one (if this invoker has them), so it's safe to modify.
+func (v *Invoker) Signers() []transaction.Signer {
+	if v.signers == nil {
+		return nil
+	}
+	var res = make([]transaction.Signer, len(v.signers))
+	copy(res, v.signers)
```

Comment:
We have `(c *Signer) Copy()`, may be it's time to use it. Do we need a deep copy here?

Reply:
We don't for accounts and likely it's not even needed there. If you think it'd be better to deep-copy signers and leave accounts, I'm OK with that. But from the practical uses no one changes signer/account internals.